package.json

```diff
@@ -72,12 +72,7 @@
     "sauce:run": "./scripts/run-with-sauce.sh",
     "test": "./tooling/index.js test",
     "test:legacy-node": "UPLOAD_DIR=node grunt test:integration",
-    "test:legacy-browser": "PACKAGE=legacy UPLOAD_DIR=browser grunt test:karma",
-    "distsrc": "sed -ibak 's#\"main\": \"dist/index.js#\"main\": \"src/index.js#' packages/node_modules/{*,@ciscospark/*}/package.json && rimraf packages/node_modules/{*,@ciscospark/*}/*bak",
```

Comment:
I believe Ian told me to remove them when I moved the eslint-config into its own module. @ianwremmel would need to confirm this.